wandb/sdk/wandb_run.py

```diff
@@ -391,6 +391,10 @@ def _make_proto_run(self, run: RunRecord) -> None:
                 run.tags.append(tag)
         if self._start_time is not None:
             run.start_time.FromSeconds(int(self._start_time))
+        if self.starting_step is not None:
+            run.starting_step = self.starting_step
+        if self.resumed is not None:
+            run.resumed = self.resumed
```

Comment:
i dont think you need this since you are changing to use the run_obj

seems more risky to unpack and repack

Reply:
Now that we are using run_obj, it appears we do not need this.